code/frameworks/angular/template/stories_angular-cli-default-ts/signal/button.component.ts

```diff
@@ -40,8 +40,7 @@ export default class SignalButtonComponent {
   /**
    * Optional click handler
    */
-  @Output()
-  onClick = new EventEmitter<Event>();
+  onClick = output<Event>();
```

Comment:
You also might want to test `output<void>()` which is totally valid 

but as far as I can see, your typings for "void" are correct as well